ckanext/data_qld/templates/base.html

```diff
@@ -1,7 +1,6 @@
 {% ckan_extends %}
 
-  {% block styles %}
-    {{ super() }}
+  {% block custom_styles %}
```

Comment:
can you explain how this changes things, should the super not be at the bottom?

Reply:
It changes things because it means that our styles are injected _after_ the call to `h.render_assets('style')` (on CKAN 2.9) and so our styles take precedence.

And we don't need `super()` because we actively block the usual sysadmin-controlled custom styling feature (which allows sysadmins to add arbitrary CSS to the site).